test/integration/forms/form.js

```diff
@@ -853,6 +855,87 @@ context('Patient Action Form', function() {
         expect(storage.isExpanded).to.be.false;
       });
   });
+
+  specify('form header widgets', function() {
+    const dob = testDateSubtract(10, 'years');
+
+    cy
+      .server()
+      .routeFormDefinition()
+      .routeActionActivity()
+      .routeFormActionFields()
+      .routeAction(fx => {
+        fx.data.id = '1';
+        fx.data.relationships.form.data = { id: '55555' };
+        fx.data.relationships['form-responses'].data = [];
+
+        return fx;
+      })
+      .routeWidgets(fx => {
+        const newWidget = getResource({
+          id: 'testFieldWidget',
+          widget_type: 'fieldWidget',
+          definition: {
+            display_name: 'Test Field',
+            field_name: 'test-field',
+          },
+        }, 'widgets');
+
+        fx.data.push(newWidget);
+
+        return fx;
+      })
+      .routePatientByAction(fx => {
+        fx.data.id = '1';
+        fx.data.attributes = {
+          first_name: 'First',
+          last_name: 'Last',
+          birth_date: dob,
+          sex: 'f',
+          status: 'active',
+        };
+
+        fx.data.relationships['patient-fields'].data = [{ id: '1', type: 'patient-fields' }];
+
+        return fx;
+      })
+      .routePatientField(fx => {
+        fx.data = {
+          id: '1',
+          type: 'patient-fields',
+          attributes: {
+            name: 'test-field',
+            value: 'Test field widget',
+          },
+        };
+
+        return fx;
+      }, 'test-field');
+
+    cy
+      .visit('/patient-action/1/form/55555')
+      .wait('@routeFormDefinition')
+      .wait('@routePatientByAction')
+      .wait('@routeAction')
+      .wait('@routeWidgets')
+      .wait('@routePatientFieldtest-field');
```

Comment:
Since we're introducing `test-field` for this test can we rename it so it's camelCase if only for this `wait`.  I don't actually think there are any hyphenated field names other than the ones in our tests, so I think it's ok to change it here.  the `-field` here is just a little awkward looking

Reply:
Makes sense, I'll change that to camelCase.